security/dirtycow.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+container_name="dirtycow"
+runtime=${RUNTIME:-kata-runtime}
+test_repo="https://github.com/dirtycow/dirtycow.github.io"
+test_dir="/root/dirtycow"
+test_file="$test_dir/test"
+test_file_content="Hello"
+dirty_file_content="pwned"
+
+finish() {
+	docker rm -f "$container_name"
+}
+trap finish EXIT
+
+# Run a gcc container
+docker run --runtime="$runtime" --name="$container_name" -dti gcc bash
+
+# Turning off periodic writeback makes exploit stable
+# This should fail inside a container
+docker exec "$container_name" bash -c "echo 0 > /proc/sys/vm/dirty_writeback_centisecs"
+[ $? == 0 ] && exit 1
```

Comment:
Might be worth displaying a message for the two failure scenarios to make it clear in the logs what happened.